packages/fxa-payments-server/src/routes/Product/index.scss

```diff
@@ -4,6 +4,7 @@
 .product-payment {
   .account-activated {
     margin-bottom: 25px;
+    margin-top: -24px;
```

Comment:
Huh, I hadn't seen this "your account is activated" screen before, glad I now know it's there, lol.

Same comment as above, in which I think we want the extra padding. Otherwise, we can change this to `margin: -24px 0 25px 0;` to make it _slightly_ better 🤷‍♀️

For reference just because I have it up and because I tagged Ryan in the last comment, without `-24px`:
<img width="710" alt="image" src="https://user-images.githubusercontent.com/13018240/69453642-f563cc80-0d29-11ea-9b68-9da0aa8b6384.png">


And with:
<img width="714" alt="image" src="https://user-images.githubusercontent.com/13018240/69453675-044a7f00-0d2a-11ea-8455-e1ad2b28b1ac.png">

Reply:
Yeah, the account activated banner was in the original UX way back when we started, but we haven't ended up using it because we could never figure out how to detect all the way through an FxA sign up that it was a brand new account. You can see it in storybook, or if you visit a product payment page with `?activated` in the query string. But, we might just remove it someday

Reply:
Yeah, the account activated banner was in the original UX way back when we started, but we haven't ended up using it because we could never figure out how to detect all the way through an FxA sign up that it was a brand new account. You can see it in storybook, or if you visit a product payment page with ?activated in the query string. But, we might just remove it someday

Reply:
Pushed an update to remove the -24px top margin